Goobi/src/de/sub/goobi/metadaten/Metadaten.java

```diff
@@ -3381,4 +3382,14 @@ public void setAddMetaDataValue(String addMetaDataValue) {
         }
         return result;
     }
+
+    /**
+     * @see @link{de.sub.goobi.forms.ProzesskopieForm.getSearchFields}
+     *
+     * @return A map containing the search fields for the currently selected OPAC
+     */
+    public HashMap<String, String> getSearchFields() {
+        return ProzesskopieForm.getSearchFieldsForCatalogue(opacKatalog);
```

Comment:
Returns an empty list if opacKatalog is still `null` (first start of meta-data editor). Fix:

    String catalogue = opacKatalog;
    if(catalogue == null) {
        List<String> allOpacCatalogues = ProzesskopieForm.getAllOpacCatalogues();
        if(!allOpacCatalogues.isEmpty()){
            catalogue = allOpacCatalogues.get(0);
        }
    }
    return ProzesskopieForm.getSearchFieldsForCatalogue(catalogue);

`ProzesskopieForm.getAllOpacCatalogues()` can and needs to be declared `static` to do so.